numpy/polynomial/tests/test_classes.py

```diff
@@ -304,6 +304,12 @@ def test_truediv(Poly):
         if not issubclass(stype, Number) or issubclass(stype, bool):
             continue
         s = stype(5)
+        if 'timedelta64' in np.asanyarray(s).dtype.name:
```

Comment:
`stype == np.timedelta64` should be enough here

Reply:
fixed